test/storage/large_garbage_collector_test.cpp

```diff
@@ -67,4 +67,115 @@ TEST_F(LargeGCTests, MixedReadWriteWithGC) {
     EndGC();
   }
 }
+
+// This test is similar to the previous one, but with a higher ration of updates
+// and longer transactions leading to more aborts.
+// NOLINTNEXTLINE
+TEST_F(LargeGCTests, HighAbortRateWithGC) {
+  const uint32_t num_iterations = 10;
+  const uint16_t max_columns = 2;
+  const uint32_t initial_table_size = 1000;
+  const uint32_t txn_length = 40;
+  const uint32_t num_txns = 1000;
+  const uint32_t batch_size = 200;
```

Comment:
Let's try to consolidate all of the shared test parameters in the test harness itself.